gensrc/proto/segment_v2.proto

```diff
@@ -164,3 +166,51 @@ message SegmentFooterPB {
     optional PagePointerPB short_key_index_page = 9;
 }
 
+message IndexPageFooterPB {
+  required int32 num_entries = 1;
```

Comment:
I think use optional to replace required is better for future.

Reply:
OK